.github/dependabot.yml

```
@@ -0,0 +1,21 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 1
+updates:
+  - package-ecosystem: "gomod"
+    directories: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)